### PR TITLE
LPS-67823 Not needed

### PR DESCRIPTION
**Pull request by brunobasto**

### CODE REVIEW

**Comment by ctampoya** [comment]
ci:retest

**Comment by brunobasto** [comment]
@fernandosouza Can you take a look at the Poshi failures?

**Comment by fernandosouza** [comment]
Sure, @brunobasto.

**Comment by ctampoya** [comment]
@fernandosouza @brunobasto I don't think the loading issue was caused by anything in this pull. 

@kenjiheigel Have you seen tests failing from this loading issue recently?

**Comment by kenjiheigel** [comment]
@ctampoya, there's been some failures recently that seem to be SPA related. I think something is behaving strangely with the timeout, but I'm not sure why. I might not have time to investigate the SPA issues though. Could we retest this one more time though? Once the start up issues are fixed at head.

**Comment by ctampoya** [comment]
@kenjiheigel I will retest once the startup errors are fixed. Thanks!

cc @brunobasto @fernandosouza

**Comment by ctampoya** [comment]
@fernandosouza After evaluating it again, I have not been able to reproduce the same loading issues happening on this pull on master at head. I was able to find the steps that causes the loading issue to consistently occur in Forms. Can you take a look at this again?

**Instance 1:**
1. Go to Forms > Click (+) to Add a New Form
2. Click on back icon to navigate back to the Form Admin page

**Instance 2:**
1. Add a New Form with a Text field
2. Save Form
3. Edit Form
4. Save Form

Result: In both instances, the page gets stuck and a slow loading message appears. The page does not navigate back until the page has been refreshed.

cc @vicnate5

File: modules/apps/forms-and-workflow/dynamic-data-lists/dynamic-data-lists-form-web/src/main/resources/META-INF/resources/admin/js/form_builder_field_settings_sidebar.js [hunk 1]
```diff
@@ -55,6 +55,20 @@ AUI.add(
...
 					},
 
+					destructor: function() {
+						var instance = this;
+
+						var toolbar = instance.get('toolbar');
+
+						toolbar.destroy();
+
+						instance.settingsForm.destroy();
```
Comment (fernandosouza) [inline modules/apps/forms-and-workflow/dynamic-data-lists/dynamic-data-lists-form-web/src/main/resources/META-INF/resources/admin/js/form_builder_field_settings_sidebar.js:65]: Hi, @brunobasto. The last @ctampoya's comment helped me to figure out the problem. The instance.settingsForm variable is only defined when the Sidebar is opened. In one of the Poshi test steps, it leaves the Form Builder page without open the Sidebar, so, this variable is never defined.

Here is my solution: https://github.com/fernandosouza/liferay-portal/blob/e80bc04d7f684e9f41b54ba1b40fc99cf357022f/modules/apps/forms-and-workflow/dynamic-data-lists/dynamic-data-lists-form-web/src/main/resources/META-INF/resources/admin/js/form_builder_field_settings_sidebar.js#L65

See the rebased commit:  https://github.com/brunobasto/liferay-portal/pull/720/commits/e80bc04d7f684e9f41b54ba1b40fc99cf357022f#L65

**Pull request closed without merging**